docs/Hosting-Packages/Overview.md

```diff
@@ -37,10 +38,9 @@ For all such purposes, NuGet supports setting up a private package source in the
 - NuGet.Server: Packages are made available through a local server through HTTP. For details, see [NuGet.Server](../hosting-packages/NuGet-Server.md).
 - NuGet Gallery: Packages are hosted on an Internet server using the [NuGet Gallery Project](https://github.com/NuGet/NuGetGallery/wiki/Hosting-the-NuGet-Gallery-Locally-in-IIS) on GitHub. NuGet Gallery provides user management and additional features such as an extensive web UI that allows searching and exploring packages from within the browser, similar to nuget.org.
 
-If you're using Visual Studio Team Services, you can also use the [Package Management extension](https://marketplace.visualstudio.com/items?itemName=ms.feed) to make NuGet packages available to your team. This extension works in conjunction with any other solution.
-
 There are also several third-party NuGet hosting products that support remote private feeds, including the following:
```

Comment:
I am updating that to read, "There are also several other NuGet hosting products that support remote private feeds..."